docs/resources/sql_table.md

```diff
@@ -196,3 +196,80 @@ This resource can be imported by its full name:
 ```bash
 terraform import databricks_sql_table.this <catalog_name>.<schema_name>.<name>
 ```
+
+## Migration from `databricks_table`
+
+The `databricks_table` resource has been deprecated in favor of `databricks_sql_table`. To migrate from `databricks_table` to `databricks_sql_table`:
+1. Define a `databricks_sql_table` resource with arguments corresponding to `databricks_table`.
+2. Add a `removed` block to remove the `databricks_table` resource without deleting the existing table by using the `lifecycle` block.
+3. Add an `import` block to add the `databricks_sql_table` resource, corresponding to the existing table.
```

Comment:
maybe this will be clearer?

```suggestion
1. Define a `databricks_sql_table` resource with arguments corresponding to `databricks_table`.
2. Add a `removed` block to remove the `databricks_table` resource without deleting the existing table by using the `lifecycle` block. If you're using Terraform version below v1.7.0, you will need to use the `terraform state rm` command instead.
3. Add an `import` block to add the `databricks_sql_table` resource, corresponding to the existing table. If you're using Terraform version below v1.5.0, you will need to use `terraform import` command instead.
```